app/schemas/collection_schema.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CollectionSchema < BaseSchema
+  def document
+    return {} unless resource.respond_to?(:draft?)
+
+    {
+      is_empty_bi: resource.draft?
```

Comment:
Just a question for my own Solr understanding — when does `is_empty_bi` get populated for all documents in Solr? Does this just magically happen or do we need to run a full reindex or something to populate this field for all collections so the existing collections with no published works don't appear in search results?

Reply:
I am open to other suggestions on how to handle it, if you have another idea.

Reply:
Thanks for confirming! I think this approach makes sense. I can check on QA to see how long it takes to index all the collections. We can do this post launch and some time during off hours if it takes a long time.

Reply:
sounds good 👍🏻